src/ralph/virtual/management/commands/openstack_sync.py

```diff
@@ -404,16 +410,18 @@ def _match_nodes_to_hosts(self, nodes):
         )
 
         for node in nodes:
+            node_sn = node.extra[self.ironic_serial_number_param]
+
             try:
                 host = CloudHost.objects.get(host_id=node.instance_uuid)
                 asset = DataCenterAsset.objects.get(
-                    sn=node.extra.get('serial_number')
+                    sn=node_sn
```

Comment:
it'd be useful to have an option to customize `DataCenterAsset` field (`sn` right now) as well